### ISSUE TITLE
Module failures when running examples notebook

### ISSUE THREAD

**Issue by wdduncan** [biolink/linkml#20]
When running the `examples.ipynb` notebook, it failed to successfully load the `as_json_obj` and the `yamlmagic` would not work correctly. Specifically:

```
import jsonasobj.as_json_obj ## ERROR: I'm getting error  No module named 'jsonasobj.as_json_obj'
```
and
```
%%yaml --loader DupCheckYamlLoader yaml

## ERROR: running this cell produces 2 errors: 
## 1. Javascript Error: require is not defined
## 2. File "<ipython-input-3-f6f890ca4330>", line 4
##    id: http://example.org/sample/example1
            ^
## SyntaxError: invalid syntax
```

I order to work around these errors I had to make a number of changes. I am attaching my notebook `exapmple-debug.ipynb` (in a zip file)
 to show how I did this. Comment marked `## ERROR:` show where I had issues.

[examples-debug.ipynb.zip](https://github.com/biolink/biolinkml/files/3905847/examples-debug.ipynb.zip)

**Comment by cmungall** [comment]
can we close this?

**Comment by wdduncan** [comment]
I don't know. Do the errors occur when running the new linkml? I'd have to run the notebook to find out ...

**Comment by wdduncan** [comment]
Cataloging errors as I attempt to run `examples.ipynb`.  
Setup before running:
- created a virtual environment: `python -m venv .env`
- installed both the `requirements.txt` and requirements-dev.txt` files using `pip`.

I get the error: `ModuleNotFoundError: No module named 'linkml.dumpers'`

![image](https://user-images.githubusercontent.com/3186638/122285229-76f6ee80-cebc-11eb-9406-5c77e386be3c.png)

**Comment by hsolbrig** [comment]
dumpers have moved to linkml-runtime.     I'd forgotten that we'd turned off the notebooks test case, so apologies for missing this.  Will turn it back on (for some reason I can't get it to work in the github actions...) and update and submit the changes.

Thanks for catching this!

**Comment by nlharris** [comment]
Can this be closed?

**Comment by cmungall** [comment]
no, the notebook needs fixed, and more broadly we need a strategy for notebooks. Either get rid of them or commit to keeping them complete and informative, use papermill + CI to check

**Comment by nlharris** [comment]
Are we keeping the notebooks?
